app/gateway/2.7.x/admin-api/consumer-groups/examples.md

```diff
@@ -0,0 +1,495 @@
+
+{% capture add_consumer %}
+{% navtabs codeblock %}
+{% navtab cURL %}
+```bash
+curl -i -X POST http://{HOSTNAME}:8001/consumer_groups/JL/consumers \
+  --data consumer=DianaPrince
+```
+{% endnavtab %}
+{% navtab HTTPie %}
+```bash
+http POST :8001/consumer_groups/JL/consumers consumer=DianaPrince
+```
+{% endnavtab %}
+{% endnavtabs %}
+{% endcapture %}
+
+{{ add_consumer | indent | replace: " </code>", "</code>" }}
+
+    Response:
+
+    ```json
+    {
+      "consumer_group": {
+      "created_at": 1638915521,
+      "id": "8a4bba3c-7f82-45f0-8121-ed4d2847c4a4",
+      "name": "JL"
+      },
+      "consumers": [
+        {
+            "created_at": 1638915577,
+            "id": "8089a0e6-1d31-4e00-bf51-5b902899b4cb",
+            "type": 0,
+            "username": "DianaPrince",
+            "username_lower": "dianaprince"
+        }
+      ]
+    }
+    ```
+
+## Set up Rate Limiting Advanced config for consumer group
+
+1. Enable the [Rate Limiting Advanced plugin](/hub/kong-inc/rate-limiting-advanced),
+configuring some basic settings:
+{% capture add_plugin %}
+{% navtabs codeblock %}
+{% navtab cURL %}
+```bash
+curl -i -X POST http://{HOSTNAME}:8001/plugins/  \
+  --data name=rate-limiting-advanced \
+  --data config.limit=5 \
+  --data config.sync_rate=-1 \
+  --data config.window_size=30 \
+  --data config.window_type=sliding \
+  --data config.retry_after_jitter_max=0 \
+  --data config.enforce_consumer_groups=true \
+  --data config.consumer_groups=JL
+```
+{% endnavtab %}
+{% navtab HTTPie %}
+```bash
+http -f :8001/plugins/  \
+  name=rate-limiting-advanced \
+  config.limit=5 \
+  config.sync_rate=-1 \
+  config.window_size=30 \
+  config.window_type=sliding \
+  config.retry_after_jitter_max=0 \
+  config.enforce_consumer_groups=true \
+  config.consumer_groups=JL
+```
+{% endnavtab %}
+{% endnavtabs %}
+{% endcapture %}
+
+{{ add_plugin | indent | replace: " </code>", "</code>" }}
+
+    For consumer groups, the following parameters are required:
+    * `config.enforce_consumer_groups=true`: enables consumer groups for this plugin.
+    * `config.consumer_groups=JL`: specifies a list of groups that this plugin allows overrides for.
+
+    {:.note}
+    > **Note:** In this example, we're configuring the plugin globally, so it will
+    apply to all entities (Services, Routes, and Consumers) in the Kong instance.
+    You can also apply it to a [specific Service or Route](/hub/kong-inc/rate-limiting-advanced)
+    for more granular control.
+
+2. The plugin you just set up applies to all consumers in the cluster. Let's
+change the settings for the `JL` consumer group only:
+
+{% capture override %}
+{% navtabs codeblock %}
+{% navtab cURL %}
+```bash
+curl -i -X PUT http://{HOSTNAME}:8001/consumer_groups/JL/overrides/plugins/  rate-limiting-advanced \
+  --header 'Content-Type: application/json' \
+  --data '{ "config": { "limit": [ 10 ], "retry_after_jitter_max": 1, "window_size": [ 10 ] } }'
```

Comment:
@yzyyi The only way I could get this one to work was with a JSON body. If I used this format (and a bunch of different syntax attempts for the arrays):

```
curl -i -X PUT http://localhost:8001/consumer_groups/JL/overrides/plugins/rate-limiting-advanced \
  -d "config.limit[1]=10" \
  -d "config.window_size[1]=10" \
```

Then I'd always get this error:
```
{"message":"2 schema violations (config.limit: {\n  \"expected a number\"\n}; config.window_size: {\n  \"expected a number\"\n})","name":"schema violation","fields":{"config":{"window_size":["expected a number"],"limit":["expected a number"]}},"code":2}%
```

Reply:
Resolved in https://github.com/Kong/kong-ee/pull/2810.